<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BG4NB0447" failures="1" tests="2" name="Test(failed)" time="26.647" errors="0" timestamp="2024-10-22T16:32:26 IST">
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="11.403">
    <failure type="java.lang.AssertionError" message="login Passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: login Passed expected [true] but found [false]
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="1.894"/>
</testsuite> <!-- Test(failed) -->
